### PR TITLE
added graphs, changed readme, added help api

### PR DESCRIPTION
**Pull request by sagarkarira**

### CODE REVIEW

File: lib/constants.js [hunk 1]
```diff
@@ -180,4 +180,88 @@ exports.countryNameMap = {
...
+
+# Source 2 stats - updated every 15 minutes from worldometers.info
+https://corona-stats.online?source=2
+
+---------------------------------------------------------------------------------
+
+# Country wise stats
+
+## Format:
+https://corona-stats.online/<countryCode>
```
Comment (XhmikosR) [inline lib/constants.js]: It seems you need to escape `<` and `>` or something @sagarkarira here and in lines 267-258

Comment (sagarkarira) [reply lib/constants.js]: I was just conveying the format. It is not an actual API. What better can be done here to convey?

Comment (XhmikosR) [reply lib/constants.js]: You need to escape the characters otherwise they don't show up in the resulting HTML :)

Comment (sagarkarira) [reply lib/constants.js]: Got it. You are right. I will fix it.

**Pull request merged**